test/unit/components/common/react_select_spec.js

```diff
@@ -49,5 +49,47 @@ describe('components/common/react_select', function() {
       });
       assert.sameMembers(listItems, ['ONE']);
     });
+
+    it('should display items with square bracket (RegExp reserved char)', function() {
```

Comment:
I'm wondering if the test cases should be increased to test also ] and ), it is probably a not real situation but as ReactSelect is a common or general component it will increase confident for future changes.